pkg/identity/service.go

```diff
@@ -209,7 +209,7 @@ func (svc *Service) Certify(ctx context.Context, req *pb.CertifyRequest) (*pb.Ce
 		}
 		var ite InvalidToken
 		if errors.As(err, &ite) {
-			log.Debugf("invalid token provided for %s: %s", reqIdentity, ite)
+			log.Infof("invalid token provided for %s: %s", reqIdentity, ite)
```

Comment:
I guess we don't care about leaking the token here because it's invalid?

Reply:
It appears `ite` doesn't reveal the token itself anyways, only error info, so no worries.